app/views/services/autoform/index.html.erb

```diff
@@ -224,8 +224,11 @@
     <div id="autoform-error" style="display: none;">
       <%= render 'services/share/flash_message', {type: :alert, message: 'Niektoré údaje ste nezadali správne. Opravte ich prosím a skúste znova.'} %>
     </div>
+    <div id="autoform-form-error" style="display: none;">
+      <%= render 'services/share/flash_message', {type: :alert, message: 'Nastala chyba pri spracovaní formuláru. Výskušajte to opäť neskôr prosím.'} %>
```

Comment:
V tomto texte by sme ešte mohli povedať,nech nám prípadne napíšu na podpora@slovensko.digital.

A druhá vec. Tento dialóg sa dá zavrieť iba raz, pretože on sa v skutočnosti zmaže jeho obsah po kliknutí na X. Viem si predstaviť, že userovi nejde internet, tak mu to raz povie, že sa nepodarilo odoslať. On ten alert zavrie a opäť tlačí odoslať a nič sa neudeje. Je to asi okrajové, ale keď už sme tu, dal by som tu namiesto toho "render ..." priamo ten kód z toho flas_message súboru a do Javascriptu hodil listener, ktorý to po stlačení Xka skryje, nevymaže.